modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityExamples.java

```diff
@@ -1109,5 +1166,806 @@ public static void main(String[] args) throws Exception {
             logger.error(String.format("Service returned status code %s: %s\nError details: %s",
                     e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
         }
+
+        try {
+            System.out.println("createProfileTemplate() result:");
+
+            // begin-create_profile_template
+            ProfileClaimRuleConditions condition = new ProfileClaimRuleConditions.Builder()
+                    .claim("blueGroups")
+                    .operator("EQUALS")
+                    .value("\"cloud-docs-dev\"")
```

Comment:
Just curious... do you really need the double-quotes embedded in the string literal itself?

Reply:
yes, these are needed, the value is encoded JSON